src/frontend/src/hooks/SessionFakeHooks.ts

```diff
@@ -0,0 +1,60 @@
+import { Guid } from 'guid-typescript';
+import { IUser } from 'types/types';
+import { useEffect, useState } from 'react';
+import useStudentContacts from './StudentContactHooks';
+
+// this is a placeholder for a 'real hook' providing the current user data (ID, name, role, etc.)
+// ideally we would actually implement this and back it with a fully fledged auth + session system
+// for now, I've mocked it with an ID and name from a real StudentContact I manually added on my machine
+// this lets me fake myself as the below user for demo purposes
+export default function useSession() {
+    // allows us to grab a real contact, which includes the current group ID (if assigned to a group)
+    const testingUserId = 'f8b20b08-7a50-4afa-994b-1bd34a7c114c';
```

Comment:
this is the critical point to set the faked user, ofc your machine will have a different GUID for a different contact but this is good enough for now